pkg/persistence/object_store.go

```diff
@@ -59,6 +60,8 @@ type BackupStore interface {
 	GetPodVolumeBackups(name string) ([]*velerov1api.PodVolumeBackup, error)
 	GetBackupContents(name string) (io.ReadCloser, error)
 	// TODO(nrb-csi): Any Get methods relevant to the CSI VolumeSnapshots should be added with the client-side PRs.
```

Comment:
I think with the 2 methods being implemented here, the comment can be deleted.

Also, I misunderstood what the `Get` methods here were being used for. They're used by the sync controller, while Velero clients use a different method to do the downloading.